File: C Language (with C++ Version)/C Fundamentals (with C++)/C Programming Operators/Logical Operators.cpp
/* Logical Operators */
// Working of logical operators

#include <iostream>
using namespace std;

int main() {
    int a = 5, b = 5, c = 10, result;
    
    //cout << fixed;
    //cout.precision(6);
    result = (a == b) && (c > b);
    cout << "(a == b) && (c > b) is " << result << endl;
    result = (a == b) && (c < b);
    cout << "(a == b) && (c < b) is " << result << endl;
    result = (a == b) || (c < b);
    cout << "(a == b) || (c < b) is " << result << endl;
    result = (a != b) || (c < b);
    cout << "(a != b) || (c < b) is " << result << endl;
    result = !(a != b);
    cout << "!(a != b) is " << result << endl;
    result = !(a == b);
    cout << "!(a == b) is " << result << endl;
    
    

    return 0;
}

File: Java/Java Fundamentals/Java Operators/Logical Operators.java
/* Logical Operators */
class Main {
  public static void main(String[] args) {

    // && operator
    System.out.println((5 > 3) && (8 > 5));  // true
    System.out.println((5 > 3) && (8 < 5));  // false

    // || operator
    System.out.println((5 < 3) || (8 > 5));  // true
    System.out.println((5 > 3) || (8 < 5));  // true
    System.out.println((5 < 3) || (8 < 5));  // false

    // ! operator
    System.out.println(!(5 == 3));  // true
    System.out.println(!(5 > 3));  // false
  }
}